sjk-core/src/main/java/org/gridkit/jvmtool/cmd/MxCmd.java

```diff
@@ -171,16 +172,19 @@ public void run() {
 			@Override
 			public void run() {
 				try {
-					if (attrib == null) {
-						host.failAndPrintUsage("MBean operation name is missing");
+					if (attribs == null || attribs.isEmpty()) {
+						host.failAndPrintUsage("MBean attribute is missing");
 					}
 					MBeanServerConnection conn = connInfo.getMServer();
                     Set<ObjectName> names = resolveSingleBean(conn);
 					MBeanHelper helper = new MBeanHelper(conn);
 					if (csv) {
+						if (attribs.size() > 0) {
+							host.failAndPrintUsage("Multiple MBean attributes not supported");
+						}
 						MTable table = new MTable();
 	                    for (ObjectName name : names) {
-						    helper.getAsTable(name, attrib, table);
+						    helper.getAsTable(name, attribs.get(0), table);
```

Comment:
Ignoring all but first attribute in case of CSV would be surprising for user.
Provided command is already capable to scrap multiple MBeans into single CSV table.
Though it may be worth to optionally add attribute name as column into CSV output.

Reply:
I handled multiple attributes in CSV as well.